aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -64,6 +64,33 @@ impl AggregatorID {
     }
 }
 
+/// Uniquely identifies each aggregator snapshot instance during the block execution.
+#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord)]
+pub struct AggregatorSnapshotID {
```

Comment:
Should we have a single ID wrapper for both snapshots and aggregators? What do you think?

Reply:
Given that we have separate data structures for aggregators and aggregator snapshots, I felt it is a cleaner to separate out the identifiers for them as well. It felt a little awkward to have `BTreeMap<AggregatorID, Aggregator>` and `BTreeMap<AggregatorID, AggregatorSnapshot>`.

Reply:
Indeed :) But when we collect aggregators and snapshots in a change set their deltas will have the same key type though, but we can do a conversion to some other type, no strong preference

Reply:
Since they are necessary to be unique against each other - i.e. AggregatorId cannot be the same as AggregatorSnapshotId, I would use the same class. If you don't like the name maybe EphemeralId for both is better?

But you can also think that it can be based on the module, and both are in 
 aggregator module

Reply:
If we use EphemeralId elsewhere, for s different class, EphemeralContextId is fine too